src/gui/widgets/FxLine.cpp

```diff
@@ -157,7 +194,9 @@ void FxLine::drawFxLine( QPainter* p, const FxLine *fxLine, bool isActive, bool
 	int width = fxLine->rect().width();
 	int height = fxLine->rect().height();
 
-	p->fillRect( fxLine->rect(), isActive ? fxLine->backgroundActive() : p->background() );
+	QColor color = muted ? mutedColor( p, fxLine, isActive ) : unmutedColor( p, fxLine, isActive );
+
+	p->fillRect( fxLine->rect(), color );
```

Comment:
Sorry to bring this up again - I can see from earlier reviews that it's already been subject to a bit of thought. The code here selects from one of four colours based on three values (`muted`, `isActive`, and the channel's `m_hasColor`). However, I feel that introducing these two methods for this overcomplicates things slightly. How about something like this:
```cpp
if (channel->m_hasColor && !muted)
{
    p->fillRect(fxLine->rect(), channel->m_color.darker(isActive ? 120 : 150));
}
else
{
    p->fillRect(fxLine->rect(), isActive ? fxLine->backgroundActive().color() : p->background().color());
}